versions.props

```diff
@@ -1,2 +1,3 @@
 com.netflix.nebula:nebula-test = 7.1.0
 com.squareup.okhttp3:* = 3.11.0
+com.google.guava:guava = 23.6.1-jre
```

Comment:
why is this required?

Reply:
errorprone itself uses some recent guava methods.  It seems that since it's happening as a javac compiler plugin (on bootclasspath), it actually overlaps with your prod classpath.

Here's the plugin we use: https://github.com/tbroyer/gradle-errorprone-plugin

cc @robert3005